### PR TITLE
Add to cart sceanrio running

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/main/java/ninjaStore/ui/pages/ShoppingCartPage.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    private ShoppingCartPage() {
+        super();
+    }
+
+    /**
+     * Gets the instance of HomePage.
+     *
+     * @return shoppingCartPageInstance.
+     */
+    public static ShoppingCartPage getInstance() {
```
Comment (Cxristian) [inline src/main/java/ninjaStore/ui/pages/ShoppingCartPage.java]: Why are your classes singleton pages?

File: src/main/java/ninjaStore/ui/pages/ShoppingCartPage.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * @version 1.0
+ */
+public class ShoppingCartPage extends BasePage {
+    private static ShoppingCartPage shoppingCartPageInstance;
+
+    @FindBy(css = "thead .text-left:nth-child(2)")
+    private WebElement tableProductNameHeader;
+
+    @FindBy(css = "tbody:nth-child(2) .text-left:nth-child(2)")
+    private WebElement firstProductName;
```
Comment (Cxristian) [inline src/main/java/ninjaStore/ui/pages/ShoppingCartPage.java]: What kind of webelement is this?? txtBox Label or Title?

File: src/main/java/ninjaStore/ui/pages/ShoppingCartPage.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/ninjaStore/ui/pages/ShoppingCartPage.java]: At first, they were singleton but now they are regular classes so they can be created according to their usage.

File: src/main/java/ninjaStore/ui/pages/ShoppingCartPage.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/ninjaStore/ui/pages/ShoppingCartPage.java]: It is the first element on a table.

**Review by Cxristian — APPROVED** [review]

**Pull request merged**